package.json

```diff
@@ -5,6 +5,7 @@
     "@types/supertest": "^6.0.2",
     "@typescript-eslint/eslint-plugin": "^7.16.1",
     "@typescript-eslint/parser": "^7.16.1",
+    "c8": "^10.1.2",
```

Comment:
who is that?

Reply:
dependency that i need to see how much of code my tests cover.
in scripts, I changed "test", adding c8 in the beginning, so that it shows a table of percentages of my code that my tests actually use.

Reply:
![image](https://github.com/user-attachments/assets/5441aa19-d004-4c13-a537-2f4036af29b4)
(low percentages are caused by velhontaloudenhoitaja)

Reply:
Cool 👍